app/javascript/topNavigation/utilities.js

```diff
@@ -52,18 +52,17 @@ export function initializeMemberMenu(memberTopMenu, menuNavButton) {
   if (navigator.userAgent === 'DEV-Native-ios') {
     document.body.classList.add('dev-ios-native-body');
   }
-
+  const { classList } = memberTopMenu;
   menuNavButton.addEventListener('click', (_event) => {
-    if (memberTopMenu.classList.contains('showing')) {
+    if (classList.contains('showing') && classList.contains('clicked')) {
       closeHeaderMenu(memberTopMenu, menuNavButton);
       menuNavButton.focus();
     } else {
       openHeaderMenu(memberTopMenu, menuNavButton);
+      classList.add('clicked');
```

Comment:
class for identifying click event

Reply:
Refactor (blocking): The CSS class provides no styling. Prefer a variable instead, e.g.

```javascript
let openedByMouse = false; // declared at the top of navigation utlitities
```

and for example, when it's clicked we set it to true.

```suggestion
	openedByMouse = true;
```

And when the member dropdown is closed, set `openedByMouse` to `false`.

Reply:
@nickytonline Good idea